### PR TITLE
Add blueprints for ROS Noetic and ROS 2 Humble

### PR DESCRIPTION
**Pull request by artivis**
This PR adds blueprints for both [ROS Noetic](http://wiki.ros.org/noetic) and [ROS 2 Humble](https://docs.ros.org/en/humble/index.html) which are the latest LTS releases of the [Robot Operating System (ROS)](https://www.ros.org/) 1 & 2.  
They both enable the upstream repo, install the project and set up the environment for the user to be ready to develop.

### CODE REVIEW

File: v1/ros-noetic.yaml [hunk 1]
```diff
@@ -0,0 +1,82 @@
+description: A development and testing environment for ROS Noetic.
+version: latest
```
Comment (townsend2010) [inline v1/ros-noetic.yaml:2]: I know many other Blueprints use `latest` here, but I'd like to get away from that convention and put in a meaningful number such as `0.1` or `1.0`, etc.

File: v1/ros2-humble.yaml [hunk 2]
```diff
@@ -0,0 +1,88 @@
+description: A development and testing environment for ROS 2 Humble.
+version: latest
```
Comment (townsend2010) [inline v1/ros2-humble.yaml:2]: idem, meaningful version

**Review by townsend2010 — COMMENTED** [review]
Hey @artivis!

We've finally got OK from Product and decided that we'll add these 2 Blueprints.  Could you please fix the small niggles mentioned below and then I think we're good.  We won't worry about the one integration test failure since that is not due to this Blueprint.

File: v1/ros-noetic.yaml (hunk 1 above)
Comment (artivis) [reply v1/ros-noetic.yaml:2]: Hi @townsend2010, just to be sure, this versionning is that of this yaml file or is it the version of the generated instance? In the former case I would go with `0.1` and increase it with subsequent changes whereas in the later case I'd mark it as `noetic` and live it as is.

Comment (townsend2010) [reply v1/ros-noetic.yaml:2]: Hi @artivis, the intent of the `version` field is for the versioning of this yaml file.

Comment (artivis) [reply v1/ros-noetic.yaml]: Thanks. I've changed it to 0.1.

File: v1/ros2-humble.yaml (hunk 2 above)
Comment (artivis) [reply v1/ros2-humble.yaml]: Changed to 0.1.

**Review by townsend2010 — APPROVED** [review]
Ok, LGTM, thanks!

**Comment by townsend2010** [comment]
bors merge

**Pull request merged**